index.js

```diff
@@ -0,0 +1,45 @@
+let weather = {
+"apiKey" : "4ff50176f17a1a9eec162b0203a0abad",
+fetchWeather: function(sity) {
+fetch("https://api.openweathermap.org/data/2.5/weather?q=" + sity + "&units=metric&appid=" + this.apiKey).then((response) =>response.json()).then((data)=>
+this.displayWeather(data));
+
+}, 
+displayWeather: function(data) {
+const{ name } =data;
```

Comment:
here you can destructure all props you need from "data" like this:
const { name, weather, main, wind } = data;
const { description } = weather[0];
const { temp, humidity } = main;
const { speed } = wind;